fv3/io/fv3atm_restart_io.F90

```diff
@@ -703,6 +703,7 @@ subroutine sfc_prop_restart_read (Sfcprop, Atm_block, Model, fv_domain, warm_sta
 
     if (global_att_exists(Sfc_restart, "file_version")) then
```

Comment:
It looks to me even with in this section, if the restart file does not have file_version, the model will still fail. Can we add an else statement to print a warning message: "WARNING: file_version is not specified, model may crash"?

Reply:
Added comment in [97c11b4](https://github.com/NOAA-EMC/fv3atm/pull/997/commits/97c11b4fdd9afb22e86effdd062628d0ba0bcbb2)

Reply:
OK, thanks. The model will not have issues with V2 restart files when the PR is merged, users need to update to the latest model version when running with v2 restart files.